ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -94,6 +98,18 @@ defaultVars() {
 			echo "Please input an architecture to test"; exit 1;;
 		*) echo "Please select a valid architecture"; showArchList; exit 1;;
 	esac
+
+	case "$OS" in
+		"debian8" | "Debian8" | "deb8" )
+			echo "debian8 selected for $ARCHITECTURE"; OS=DEBIAN8;;
+		"debian10" | "Debian10" | "deb10" )
+			echo "debian10 selected for $ARCHITECURE"; OS=DEBIAN10;;
+		"ubuntu18" | "u18" | "Ubuntu18" )
+			echo "ubuntu18 selected for $ARCHITECURE"; OS=UBUNTU18;;
```

Comment:
Sorry to be nitpick-y but when `echo`ing to the user, could you capitalise the distribution?